akka-cluster/src/main/scala/akka/cluster/Cluster.scala

```diff
@@ -356,6 +356,15 @@ class Cluster(val system: ExtendedActorSystem) extends Extension {
     else
       clusterDaemons ! InternalClusterAction.AddOnMemberRemovedListener(callback)
   }
+
+  /**
+   * Generate the actor path by replacing the Address in the RootActor Path for the given
+   * ActorRef with the cluster's `selfAddress`, unless an address is already included
+   */
+  def pathOf(actorRef: ActorRef): ActorPath = {
+    ActorPath.fromString(actorRef.path.toStringWithAddress(selfAddress))
+  }
```

Comment:
sorry, I didn't think about the purpose of this utility and the name did not guide me.
We should name it something like `fillInMissingAddress`.
I think the parameter should be a `ActorPath` instead of an `ActorRef`.